5.1.3/httpd.conf

```diff
@@ -2,13 +2,13 @@
 # This is the main Apache HTTP server configuration file.  It contains the
 # configuration directives that give the server its instructions.
 # See <URL:http://httpd.apache.org/docs/2.2> for detailed information.
-# In particular, see
+# In particular, see 
```

Comment:
When I tried to run the Image, the apache2 2.2.32 won't start with the old `http.conf`.